Debian/8/Makefile

```diff
@@ -36,6 +37,9 @@ ifeq ($(OVAL_5_11), 0)
 	# Update requested debian8 OVAL version in $(CONF)/oval.config to be the 5.11 OVAL language version
 	sed -i "s/^\(debian8_oval_version = \).*/\15.11/" $(CONF)/oval\.config
 endif
+# 	if needed, list found orphans
+#	$(LOCAL_UTILS)/find_orphans.py $(PROD)|sort
+	$(LOCAL_UTILS)/find_orphans.py $(PROD) | xargs rm
```

Comment:
This script, and this option, actually breaks the entire purpose of `make validate` and should not be added. It would be better to the following:
- Comment out the Debian `make validate` from the global Makefile for a time being
- Comment out the `<oval>` tags in the xccdf content.

In addition, checks like `installed_OS_is_centos6.xml` maybe valid orphan file from xccdf content and may only be called from the oval content itself.